spec.md

```diff
@@ -49,36 +49,35 @@ Every Span has zero or more **Logs**, each of which being a timestamped event na
 
-Trace Attributes come with powerful _costs_ as well; since the attributes are propagated in-band, if they are too large or too numerous they may reduce system throughput or contribute to RPC latency.
+Trace Attributes come with powerful _costs_ as well; since the attributes are propagated in-band, if they are too large or too numerous they may decrease system throughput or increase RPC latencies.
 
 **Trace Attributes** vs. **Span Tags**
 
-* Trace Attributes are propagated in-band (i.e., alongside the actual application data) across process boundaries. Span Tags are not propagated since they are not inherited from parent Span to child Span.
-* Span Tags are recorded out-of-band from the application data, presumably in the tracing system's storage. Trace Attributes are not necessarily recorded here, though an implementation may elect to.
+- Trace Attributes are propagated in-band (i.e., alongside the actual application data) across process boundaries. Span Tags are not propagated since they are not inherited from parent Span to child Span.
+- Span Tags are recorded out-of-band from the application data, presumably in the tracing system's storage. Trace Attributes are not necessarily recorded there, though an implementation may elect to.
 
 Also, trace attribute keys have a restricted format: implementations may wish to use them as HTTP header keys (or key suffixes), and of course HTTP headers are case insensitive. As such, trace attribute keys MUST match the regular expression `(?i:[a-z0-9][-a-z0-9]*)`, and – per the `?i:` – they are case-insensitive. That is, the trace attribute key must start with a letter or number, and the remaining characters must be letters, numbers, or hyphens.
 
 
-## Platform-Independent API Semantics
+# Platform-Independent API Semantics
 
-OpenTracing supports a number of different platforms, and of course the per-platform APIs try to blend in to their surroundings and do as the Romans do. That said, each platform API must model a common set of semantics for the core tracing concepts described above. In this document we attempt to describe those concepts and semantics in a language- and platform-agnostic fashion.
+OpenTracing supports a number of different platforms, and of course the per-platform APIs try to blend into their surroundings and "do as the Romans do." That said, each platform API must model a common set of semantics for the core tracing concepts described above. In this document we attempt to describe those concepts and semantics in a language- and platform-agnostic fashion.
```

Comment:
Late note, but non-english might struggle to unravel blending and doing as Romans bit. 

Maybe be more explicit: OT apis try to adhere to idioms and conventions of their language and platforms.

Feel free to punt if distracting